lib/matplotlib/backends/backend_svg.py

```diff
@@ -26,196 +26,120 @@
 from matplotlib.transforms import Affine2D, Affine2DBase
 from matplotlib import _png
 
-from xml.sax.saxutils import escape as escape_xml_text
+from xml.etree import ElementTree as etree
 
 backend_version = __version__
 
-# ----------------------------------------------------------------------
-# SimpleXMLWriter class
```

Comment:
Please leave this license unless you really have a 100% clean-room re-write (which I do not think this is as there is one method which is identical).  I err on the side of caution in these matters.